src/resolvers/avocado.ts

```diff
@@ -37,18 +37,17 @@ const AvoConnectorMapping: Record<ChainId, Record<string, string>> = {
 }
 
 const AvoMultiPaybackMapping: Record<ChainId, string> = {
-    1: "0xb03922E93c386B045b2a7f7e6732Ef4F3B93f993",
-    137: "",
-    42161: "0x62750CfEbDB196CAe6DD6956c13AD60682a614d0",
-    43114: "",
-    10: "",
+    1: "0xb03922E93c386B045b2a7f7e6732Ef4F3B93f993", // Mainnet
+    137: "0x5C122207f668D3fE345465Ac447b3FEF627f4963", // Polygon
+    42161: "0x62750CfEbDB196CAe6DD6956c13AD60682a614d0", // Arbitrum
+    43114: "0xFe0ccFac4B1502259Bb2359e25f6bC732De93c56", // Avalanche
+    10: "0x1b5687b3132a88120Cf41B58d6d454A6015ea0a2", // Optimism
     250: "",
```

Comment:
@shriyatyagii can you throw proper error?

Reply:
Added error log